cpp/src/gandiva/function_registry_datetime.cc

```diff
@@ -83,6 +83,38 @@ std::vector<NativeFunction> GetDateTimeFunctionRegistry() {
       NativeFunction("castBIGINT", {}, DataTypeVector{day_time_interval()}, int64(),
                      kResultNullIfNull, "castBIGINT_daytimeinterval"),
 
+      NativeFunction("castINT", {}, DataTypeVector{month_interval()}, int32(),
```

Comment:
why not use castINT and castNULLABLEINT as alias

Reply:
fixed